recipes/cudadecon/yum_requirements.txt

```diff
@@ -0,0 +1,2 @@
+wget
+perl
```

Comment:
Can these be included in `build` instead?

Reply:
indeed they can